beacon-chain/sync/validate_beacon_blocks.go

```diff
@@ -236,7 +232,12 @@ func (s *Service) validateBeaconBlock(ctx context.Context, blk interfaces.ReadOn
 	ctx, span := trace.StartSpan(ctx, "sync.validateBeaconBlock")
 	defer span.End()
 
-	if err := validateDenebBeaconBlock(blk.Block()); err != nil {
+	if err := s.validateEPBSBeaconBlock(blk.Block()); err != nil {
+		s.setBadBlock(ctx, blockRoot)
+		return err
+	}
+
+	if err := s.validateDenebBeaconBlock(blk.Block()); err != nil {
```

Comment:
The  Deneb validation cannot come unconditionally after an ePBS validation, these blocks are validated differently.